zookeeper-server/pom.xml

```diff
@@ -87,6 +87,10 @@
       <groupId>ch.qos.logback</groupId>
       <artifactId>logback-classic</artifactId>
     </dependency>
+    <dependency>
+      <groupId>co.elastic.logging</groupId>
+      <artifactId>logback-ecs-encoder</artifactId>
+    </dependency>
```

Comment:
While this might be useful for some... most people are probably not going to use this. It doesn't make sense to add this optional dependency to the class path for everybody. Some people may want a more minimal approach, for a smaller security footprint, and to save space in their downstream environment.